operator/apis/platform/v1alpha1/central_types.go

```diff
@@ -115,9 +115,8 @@ type CentralComponentSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=5
 	Persistence *Persistence `json:"persistence,omitempty"`
```

Comment:
It would be great to enumerate what actually is stored on this directly attached PVC (I think someone mentioned it's still used "for some things" but unclear for what). As a customer I would be confused by both `persistence` and `db` mentioning persistent data.

Reply:
You are right. Even though it's a small effort I would prefer to do it in the separate PR. I need to gather info about its usage